test/recipes/70-test_quic_multistream.t

```diff
@@ -43,3 +44,18 @@ SKIP: {
                "running qlog verification script");
     };
 }
+
+SKIP: {
+    skip "no qlog", 1 if disabled('qlog');
+    skip "not running CI tests", 1 unless $ENV{OSSL_RUN_CI_TESTS};
+    skip "not running artifacts upload", 1 unless $ENV{OSSL_CI_ARTIFACTS_PATH};
+
+    subtest "copy qlog artifacts to upload directory" => sub {
+        plan tests => 1;
+
+        my $artifacts_path = $ENV{OSSL_CI_ARTIFACTS_PATH};
+        mkpath("${artifacts_path}/quic_multistream_test");
+        ok(run(cmd(["mv", "--", $qlog_output,
+                    "${artifacts_path}/quic_multistream_test/"])));
+    };
+}
```

Comment:
This strikes me as very odd.  I would rather have the caller (i.e. the CI) be able to set the environment variable `QLOGDIR`, and let the caller do the rsync.

You could have this at the start of the recipe:

```
$ENV{QLOGDIR} = result_dir("qlog-output") unless $ENV{QLOGDIR};
```

and then go from there.

Reply:
Hmm... but how do we get `result_dir()`? That is computed within the test framework, and it's where unit test-specific output is supposed to go.